views/base.jade

```diff
@@ -18,6 +18,14 @@ html(lang="en")
     meta(charset='utf-8')
     meta(name="viewport", content="width=device-width, initial-scale=1")
     link(rel="stylesheet", href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css")
+    script.
+      window.gapiReady = new Promise(resolve => {
```

Comment:
Kinda horrible but I think we need something like this if loading the `gapi` library async.